CorsixTH/Src/th_gfx_sdl.cpp

```diff
@@ -443,6 +443,12 @@ void THRenderTarget::setBlueFilterActive(bool bActivate)
     m_bBlueFilterActive = bActivate;
 }
 
+// Actiate and Deactivate SDL function to capture mouse to window
+void THRenderTarget::setWindowGrab(bool bActivate)
+{
+    SDL_SetWindowGrab(m_pWindow, (SDL_bool)(bActivate ? 1: 0));
```

Comment:
Prefer c++ style casts.
In this case though use `SDL_TRUE` and `SDL_FALSE` in the expression instead of casting.